comparison.md

```diff
@@ -442,7 +442,7 @@ Help improve our documentation! If you find anything wrong or which could be imp
 			<tr>
 				<td><strong>Support for external (Amazon S3, Dropbox, etc.) backups via “backup and migrate” module</strong></td>
 				<td>✅ Full support to save backups externally. Custom crons provide even more flexibility.</td>
-				<td>⚠️ “backup and migrate” module not recommended</td>
+				<td>⚠️ Backups made to S3 automatically. “backup and migrate” module not recommended</td>
```

Comment:
@stevector But this is the S3 account of Pantheon? So if Pantheon cannot pay the bills anymore (see Aberdeen), the client has no access to the backups?